client/css/style.css

```diff
@@ -1643,11 +1643,16 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	opacity: 1;
 }
 
-.textcomplete-item .emoji {
-	margin-right: 8px;
-	width: 16px;
-	text-align: center;
+.inline-emoji {
 	display: inline-block;
+	font-size: 1.4em;
```

Comment:
I think 1.4 is the limit here before it starts pushing the lines around, but I will try again.